packages/arcgis-rest-auth/test/UserSession.test.ts

```diff
@@ -1839,4 +1920,327 @@ describe("UserSession", () => {
         });
     });
   });
+
+  describe(".getPortal()", () => {
+    afterEach(fetchMock.restore);
+
+    it("should cache metadata about the portal", (done) => {
+      // we intentionally only mock one response
+      fetchMock.once(
+        "https://www.arcgis.com/sharing/rest/portals/self?f=json&token=token",
+        {
+          fetchAuthorizedDomains: ["gis.city.com"],
```

Comment:
Wrong variable name here and in a few other places:
```suggestion
          authorizedCrossOriginDomains: ["gis.city.com"],
```